content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,262 @@
+
+### Install and run the Datadog development toolkit
+
+The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are developing a data integration by generating a skeleton of your integration tile's assets and metadata.
+
+First, ensure that [Python v3.8 and later][17] is installed. To avoid potential environment conflicts, create a virtual environment in the directory where you cloned the Marketplace repository by running `python3 -m pip install virtualenv --user`.
+
+Then, name and activate the virtual environment with these commands:
+
+```python
+virtualenv [name of env]
+source [name of env]/bin/activate
+```
+
+To install the latest released version of the Datadog Development Toolkit from [PyPI][18], run `python -m pip install --upgrade "datadog-checks-dev[cli]"`.
+
+If you are using the Z Shell, you may need to use escaped characters by running `python -m pip install --upgrade datadog-checks-dev\[cli\]`.
+
+To set the location of the cloned repository directory: 
+
+```python
+ddev config set marketplace [/path/to/marketplace_directory]
+ddev config set repo marketplace
+```
+
+### Create an informational tile only listing
+
+For standalone SaaS licenses, Datadog Apps, professional services, and Datadog REST API-based integrations, the Datadog Development Toolkit creates the scaffolding for an informational tile-only listing. To create the informational tile-only listing's scaffolding, run `ddev create -t tile -v2 "<Offering Name>"` in the `marketplace` directory specified above.
+
+### Create a full data integration
+
+To generate a complete data integration's scaffolding, run `ddev create -v2 "<Offering Name>"` from the `marketplace` directory specified above.
+
+### Populate the tile scaffolding
+
+The `ddev` command generates a skeleton of the following folders and files belonging to your integration tile's assets.
+
+{{% integration-assets %}}
+
+#### README
+
+After creating a `README.md` file, follow these instructions to complete your listing:
+
+1. Write a description under an `## Overview` header that describes the value your offering provides to users and benefits to purchasing and installing the integration in the Datadog Marketplace (for example, out-of-the-box dashboards for more comprehensive observability, replays of user sessions, logs, alerts, and more). This information is displayed in the **Overview** tab on the integration tile.
+2. Include all the steps to setting up your Marketplace integration under a `##Setup` header that includes information divided into H3 headings (`###`) about installing the integration using the in-app integration tile, configuring the integration with the appropriate roles and permissions in your Datadog organization, out-of-the-box Datadog features that users who purchased and installed the integration can access (such as metrics, events, monitors, logs, dashboards, and more), and uninstalling the integration.
```

Comment:
Can we update this to include the uninstallation changes we're adding?

Reply:
Sure, I'm gonna make this into a table so it's easier to read